docroot/modules/custom/ccom_core/ccom_core.module

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * Contains ccom_core.module.
+ */
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function ccom_core_preprocess_block(&$variables) {
+  if (isset($variables['attributes']['id']) && $variables['attributes']['id'] === 'block-uids-base-footercontactinfo') {
+    $current_path = \Drupal::request()->getRequestUri();
+    $full_url = \Drupal::request()->server->get('HTTP_HOST') . $current_path;
+    $newlink = '<p><a href="http://hc-vbugtracker.healthcare.uiowa.edu/projects/uihc-org/issues/new?issue[custom_field_values][1]=' . $full_url . '">Report an issue with this page</a></p>';
+    $variables["content"]["field_uiowa_text_area"][0]['#text'] .= $newlink;
+    $variables["content"]["field_uiowa_text_area"][0]['#cache'] = [
```

Comment:
```suggestion
    $variables['#cache'] = [
```

This should be set on the block of `preprocess_block`, not the render array within

Reply:
This has resolved caching issues for me when browsing anonymously and switching between pages and the link not appearing.